lib/matplotlib/mpl-data/matplotlibrc

```diff
@@ -33,10 +33,14 @@
 ## Colors: for the color values below, you can either use
 ##     - a Matplotlib color string, such as r, k, or b
 ##     - an RGB tuple, such as (1.0, 0.5, 0.0)
-##     - a hex string, such as ff00ff
+##     - a hex string, such as "#ff00ff" (for backward compatibility, but
```

Comment:
It took me a second reading to realize that "for backward compatibility" didn't refer to "#ff00ff". Maybe it is better to start a new sentence with that information? Or something like "(also, for backwards compatibility, ff00ff is supported, but discouraged)".